src/main/java/io/confluent/connect/hdfs/parquet/ParquetRecordWriterProvider.java

```diff
@@ -49,7 +49,7 @@ public String getExtension() {
 
     Path path = new Path(fileName);
     final ParquetWriter<GenericRecord> writer =
-        new AvroParquetWriter<>(path, avroSchema, compressionCodecName, blockSize, pageSize);
+        new AvroParquetWriter<>(path, avroSchema, compressionCodecName, blockSize, pageSize, true, conf);
```

Comment:
I think it is fine to enable dictionary for now. Also this is in consistent with the current behavior. In the long term, we need to add more configurations to make the parquet writer more configurable.